airflow/models/xcom.py

```diff
@@ -620,32 +621,41 @@ def serialize_value(
         if conf.getboolean("core", "enable_xcom_pickling"):
             return pickle.dumps(value)
         try:
-            return json.dumps(value).encode("UTF-8")
-        except (ValueError, TypeError):
+            return json.dumps(value, cls=XComEncoder).encode("UTF-8")
+        except (ValueError, TypeError) as ex:
             log.error(
-                "Could not serialize the XCom value into JSON."
+                f"{ex}."
```

Comment:
```suggestion
                "%s."
```

Reply:
That seems to go against the current style?

Reply:
I'm surprised this wasn't triggering lint issues -- the rule we use is to never use f-string in log messages.

Reply:
```
      - id: check-lazy-logging
        name: Check that all logging methods are lazy
        entry: ./scripts/ci/pre_commit/pre_commit_check_lazy_logging.py
```

Reply:
Okido :-P geez :-)